### PR TITLE
Example for the selection of intervals in DimArrays added to the course.

### PR DESCRIPTION
**Pull request by dpabon**

### CODE REVIEW

**Comment by rafaqz** [comment]
Thanks!

The recommended way to do that for `Dim{:x}` dimensions is:
```julia
A2[distance=12 .. 35, time=Date(2001, 5) .. Date(2001, 7))]
```

**Comment by dpabon** [comment]
Great! Thanks for the clarification!

I could not find that way in the documentation. Shall I close the PR and open a new one?

**Comment by rafaqz** [comment]
You can also just edit this PR inline

**Comment by rafaqz** [comment]
The comment above now doesnt match the example ;)

**Comment by dpabon** [comment]
Ups… Thanks for the catch! Now should be correct.

File: docs/crash/course/course.jl [hunk 1]
```diff
@@ -150,6 +150,9 @@ mean(A3; dims=Ti)
 
 A[X(12..35), Ti(Date(2001, 5)..Date(2001, 7))]
 
+# To select intervals in DimArrays (e.g. ```A2```) you need to specify ```dimname=a .. b```:
+
+A2[distance=12 .. 35, time=Date(2001, 5) .. Date(2001, 7))]
```
Comment (rafaqz) [inline docs/crash/course/course.jl:155]:
```suggestion
A2[distance=12 .. 35, time=Date(2001, 5) .. Date(2001, 7)]
```

**Pull request merged**